.github/workflows/ci.yml

```
@@ -94,6 +94,7 @@ jobs:
     needs:
       - rustfmt
       - check-release
+    secrets: inherit 
```

Is this line required? There are no changes in the `github-release.yml` file regarding the secrets, but we do have the token secret used within the file. I'm not sure on this, but https://github.com/stacks-network/stacks-core/pull/5976/files removed it and it seemed to work well without it

---

sadly, it's required - the problem was that with 5976, the docker images were not being published (docker login was silently failing in composite action). in this PR, it caused the attestation steps to also fail. 
https://github.com/stacks-network/stacks-core/actions/runs/14246362302/job/39933810440
was a test run using 5976, and the docker images were not published (even though everything was built).

---

Understood, thank you for clearing that up!

---

adding for clarity from a test run on an rc using #5976 : https://github.com/stacks-network/stacks-core/actions/runs/14246362302/job/39933810440#step:2:236
```
...
  env:
    BRANCH_NAME: release/3.1.0.0.8-rc1
    GITHUB_SHA_SHORT: 405f779
    GITHUB_REF_SHORT: release/3.1.0.0.8-rc1
    DOCKER_PUSH: false
```

`DOCKER_PUSH` here should have been true, that's what clued me in that there was a problem with the auth step.